src/app/tabs/settings-tab.js

```diff
@@ -10,14 +10,15 @@ import * as packageJson from '../../../package.json'
 const profile = {
   name: 'settings',
   displayName: 'Settings',
-  methods: [],
+  methods: ['getGithubAccessToken'],
```

Comment:
Why "Access" ?

Reply:
that's `Personal access tokens` in https://github.com/settings/tokens